### PR TITLE
Add make install kafka

### PR DESCRIPTION
**Pull request by skonto**
- add commands to setup Knative Kafka and Strimzi
- update docs
/cc @matzew @aliok @markusthoemmes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,15 @@ install-serving:
 install-eventing:
 	INSTALL_SERVING="false" ./hack/install.sh
 
+install-kafka:
+	INSTALL_SERVING="false" INSTALL_KAFKA="true" ./hack/install.sh
```
Comment (skonto) [inline Makefile:19]: By default eventing, serving are being installed. Knative Kafka sets Eventing as a hard dependency so we need to enable the latter by default.

**Comment by skonto** [comment]
Tests dont pass although I didnt touch anything.

File: Makefile (hunk 1 above)
Comment (matzew) [reply Makefile:19]: should we have an „install-all“ ?

including serving and strimzi?

**Comment by aliok** [comment]
@skonto That's a flake. Sometimes happens unfortunately.

I think the changes look good. We had the env vars enabled/disabled for tests but there were nothing for devl.

/lgtm
/approve
/hold

Leaving the unhold to @matzew as I saw he had a question.

File: Makefile (hunk 1 above)
Comment (skonto) [reply Makefile:19]: Makes sense to me to have just one command to launch everything. I will added it.

**Comment by skonto** [comment]
@matzew I added `make install-all`.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**